Resources/Server Info/Rules.txt

```diff
@@ -1,85 +1,95 @@
-[color=#ff0000]YOU MUST BE AT LEAST 16 YEARS OF AGE TO PLAY ON WIZARD'S DEN SERVERS. ANY USER SUSPECTED OF BEING UNDER 16 YEARS OF AGE WILL BE BANNED UNTIL THEY ARE OF AGE.[/color]
+[color=#ff0000]YOU MUST BE AT LEAST 16 YEARS OF AGE TO PLAY ON WIZARD'S DEN SERVERS. ANY USERS SUSPECTED OF BEING UNDER 16 YEARS OF AGE WILL BE BANNED UNTIL THEY ARE OF AGE.[/color]
```

Comment:
Shouldn't this be singular like it was before?

Reply:
It works singular or plural.

Reply:
As for rule numbering I think that just makes a different numbering problem. I personally don't think the numbering is exactly important except for a reference how far down a rule is which this is accomplished by the current scheme.

Reply:
> It works singular or plural.

Then don't change it??

> As for rule numbering I think that just makes a different numbering problem.

What problem?

Reply:
> Then don't change it??
I like it plural.
> What problem?
I would have to go and change the rule numbers for everything and ensure they are assigned to the correct arbitrary section for little gain. I think the rule numbers work the way they are just fine, so like you said if it works both ways I probably shouldn't change it.